frontend/views/components/modal/Modal.vue

```diff
@@ -110,7 +110,8 @@ export default {
         this.content = null
         this.contentProps = {}
         // Refocus on button that open the modal
-        this.lastFocus.focus()
+        // TODO/BUG It might not exist... (e.g. profile card was closed). What's the best approach then?
```

Comment:
The focus should be on the button that open the modal. In case of modal opening a modal, meaning that the last focus element is not present, the best approach I guess is to focus on the button that open the first modal. We can do that by recording the previous focused element if the last one doesn't exist. Or a simpler solution is to go back to the first focusable element in the page

Reply:
Here the trick is the profile|notifications cards. For example, clicking "Remove Member" in the profile card, it opens the modal and closes the card. This means that `this.lastFocus` does not exist anymore. When we close the modal, it will go back to the top of the page because of that. But it should go back to the respective user in the members' list. 
I didn't find any clean way to fix this... but I'll keep in the back of my head